pkg/virt-handler/vm_test.go

```diff
@@ -1465,66 +1471,200 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 	})
 	Context("When VirtualMachineInstance is connected to a network", func() {
-		It("should report the status of this network", func() {
-			vmi := NewPendingVirtualMachine("testvmi")
-			vmi.Status.Phase = v1.Scheduling
-			pod := NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
 
-			addVirtualMachine(vmi)
-			podFeeder.Add(pod)
+		It("should only report the pod network in status", func() {
 
-			networkName := "test net"
-			podIp := "1.1.1.1"
-			pod.Status.PodIP = podIp
+			vmi := v1.NewMinimalVMI("testvmi")
+			vmi.UID = vmiTestUUID
+			vmi.ObjectMeta.ResourceVersion = "1"
+			interface_name := "interface_name"
 			vmi.Spec.Networks = []v1.Network{
-				v1.Network{
-					Name: networkName,
-					NetworkSource: v1.NetworkSource{
-						Pod: &v1.PodNetwork{
-							VMNetworkCIDR: "1.1.1.1",
+				{
+					Name:          interface_name,
+					NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}},
+				},
+			}
+			vmi.Status.Phase = v1.Running
+			vmi.Status.Interfaces = make([]v1.VirtualMachineInstanceNetworkInterface, 0)
+			vmi = addActivePods(vmi, podTestUUID, host)
+
+			podCacheInterface := network.PodCacheInterface{
+				Iface: &v1.Interface{
+					Name: interface_name,
+				},
+				PodIP: "1.1.1.1",
+			}
+			podJson, err := json.Marshal(podCacheInterface)
+			Expect(err).ToNot(HaveOccurred())
+			err = os.MkdirAll(fmt.Sprintf(util.VMIInterfaceDir, vmi.UID), 0755)
+			Expect(err).ToNot(HaveOccurred())
+			vmiInterfacepath := fmt.Sprintf(util.VMIInterfacepath, vmi.UID, interface_name)
+			f, err := os.Create(vmiInterfacepath)
+			Expect(err).ToNot(HaveOccurred())
+			_, err = f.WriteString(string(podJson))
+			Expect(err).ToNot(HaveOccurred())
+			f.Close()
+
+			updatedVMI := vmi.DeepCopy()
+			updatedVMI.Status.Phase = v1.Running
```

Comment:
Seems redundant. Wasn't it already [set](https://github.com/kubevirt/kubevirt/pull/3488/commits/32e0cabfe7c7048a6e2147cbadfc8cb296647312#diff-14837489e428608f8ed1f61f9d7242bcR1487) on the vmi and passed to the updatedVmi via the copy?

Reply:
Removed now